scripts/generate-linux-digests.sh

```diff
@@ -9,14 +11,18 @@ if [ -z "${DRONE_TAG}" ]; then
     exit 0
 fi
 
+skopeoinspect() {
+    skopeo inspect --retry-times 15 "$@"
```

Comment:
Should we reduce the retry times to maybe 10 or a value that doesn't exceed the Drone build timeout? as suggested above. So the script fails before the build times out? Otherwise, lgtm

Reply:
If the Drone timeout is leading, then we can only retry 2 times to allow all images to be retried (696 images in the v2.7.7).

The trade-off is wether you want all images to be able to retry within the hour (which limits the amount of retries) or to be able to be resilient to an outage where one or more images can take longer.

Reply:
Oh man, ok. Increase the Drone timeout and keep retries at 15 to stay resilient for outages where 'heavy' images need to be pulled again?
If Drone fails due a timeout, will there be messages in the CI outlining the image pull failures?

Reply:
See the output in https://github.com/rancher/rancher/pull/43116#issue-1934449863

Reply:
That's fine, thanks for answering my questions!